sonaranalyzer-dotnet/its/expected/Nancy/Nancy.Tests-{776D244D-BC4D-4BBB-A478-CAF2D04520E1}-S3966.json

```diff
@@ -1,17 +0,0 @@
-{
-"issues":  [
-{
-"id":  "S3966",
-"message":  "Refactor this code to make sure 'innerStream' is disposed only once.",
```

Comment:
was this an instance of the StreamWriter problem?

Reply:
Yes.

```
using (var innerStream = new MemoryStream())
using (var streamWriter = new StreamWriter(innerStream, Encoding.UTF8) { AutoFlush = true })
```